lib/service/forum/ForumMeldingenService.php

```diff
@@ -207,6 +254,50 @@ public function getDraadMeldingNiveauVoorLid(ForumDraad $draad, $uid = null)
 		}
 	}
 
+	/**
+	 * Laad push bericht
+	 *
+	 * @param Account $ontvanger
+	 * @param Profiel $auteur
+	 * @param ForumPost $post
+	 * @param ForumDraad $draad
+	 * @throws LoaderError
+	 * @throws RuntimeError
+	 * @throws SyntaxError
+	 */
+	private function laadPushBericht(
```

Comment:
Misschien om aan te geven dat er een side-effect is

```suggestion
	private function stuurPushBericht(
```